src/runner/GenericFunctionProfiler.cpp

```diff
@@ -18,6 +18,8 @@ namespace runner {
             throw std::runtime_error("Could not find binary at " + execPath);
         }
 
-        system(execPath.c_str());
+        int error = system(execPath.c_str());
+        if (error != 0)
```

Comment:
Convention in this project is to put curly brackets with `if`s even if they're one liners